tasks/main.yml

```diff
@@ -41,62 +41,52 @@
     name: "{{ __mssql_server_packages }}"
     state: present
 
-- name: Check if the errorlogfile setting has a custom value
-  command: /opt/mssql/bin/mssql-conf get filelocation errorlogfile
-  register: __mssql_errorlogfile
-  ignore_errors: true
+- name: Check if the errorlog file exists and its location
+  shell: |
+    set -euo pipefail
+    errorlog="$(/opt/mssql/bin/mssql-conf get filelocation errorlogfile 2>&1 \
+    | sed 's\errorlogfile : \\')" || :
```

Comment:
Use `/` instead of `\` in the `sed` command - the `\` is typically used for escaping instead of a command delimiter.